### PR TITLE
General documentation fixes.

### LINKED ISSUE #298: Documentation changes
Google Cloud Setup Section: 
- [x] Step 2: Kubernetes. Needs more documentation. Steps 1: User has to enable APIs. Step 2: Search for kubernetes, click enable.  Step 3: Add credentials to your project. Select "Yes" to question 1. After that I got the following message: "You don't need to create new credentials.
For your situation you can use Application Default Credentials, which provide a simple way to access Google APIs from App Engine or Compute Engine. Learn how to use Application Default Credentials. If you prefer, you can create a service account: https://developers.google.com/identity/protocols/application-default-credentials?hl=en_US
https://console.cloud.google.com/iam-admin/serviceaccounts?project=deepcellsoftware&folder&organizationId
I got super lost at this point, not sure how to proceed. Luckily our main project already has kubernetes enabled, so I went ahead on that one. 

- [x] Step 4: GPU quotas. I don't have an option for GPUs (all regions). I have to set quotas for individual GPUs. e.g., our GCP instance has a quota of 4 NVIDIA V100 GPUs specifically. I think more explicit instructions here would be helpful. "From the 'Metrics' dropdown menu, select NVIDIA V100 GPU and request at least 1." 

- [x] Step 4: IP address quotas. Same thing for IP Addresses: "From the 'Metrics' dropdown menu, scroll to 'In-use IP Addresses' and make sure the limit is at least 16. 

- [x] Step 5: Create a bucket. I would specify that the user should select Standard storage class and fine-grained access control. I don't think the second one matters, but the less guessing during installation the better. 

Starting the Kiosk:

- [x] For the note that starts "This command and the one that follows may need..."  it could be useful to also include a comment about needing to supply a password 

Deecell Kiosk Usage:

- [x] "select the configuration option for your chosen cloud provider" It isn't obvious that Configure GKE is the actual command I need to select; I'd put this in the description as well. 

- [x]  "Name your cluster." Because the project and bucket are pre-specified names, it's not obvious that the cluster is being created here, rather than referencing something else that already exists like for the other two options

- [x] Bug with firewalls / account upgrade

One fix for firewall issue is pushed I'll continue!

### LINKED ISSUE #363: Docker run command in readthedocs
Docker run command in https://deepcell-kiosk.readthedocs.io/en/master/GETTING_STARTED.html is: 
docker run -e DOCKER_TAG=1.2.0 vanvalenlab/kiosk:1.2.0 | sudo bash 

but might need to be 
docker run -e DOCKER_TAG=1.2.0 vanvalenlab/kiosk-console:1.2.0 | sudo bash

**To Reproduce**
1. Run 'docker run -e DOCKER_TAG=1.2.0 vanvalenlab/kiosk:1.2.0 | sudo bash'
2. Enter 'n/a'
3. See error
"Unable to find image 'vanvalenlab/kiosk:1.2.0' locally
docker: Error response from daemon: manifest for vanvalenlab/kiosk:1.2.0 not found."

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [e.g. macOS, Ubuntu]

**Additional context**

Additionally, the command to run the kiosk-console in the docs is just 'kiosk' but it should be 'kiosk-console'

### PR DESCRIPTION
**Pull request by willgraf**
While going through a test run with a new account, I found some bits of the documentation could use touchups.

[The docs are built on RTD for review as well](https://deepcell-kiosk.readthedocs.io/en/docs/).

Fixes #298 and fixes #363

### CODE REVIEW

File: docs/source/GETTING_STARTED.rst [hunk 1]
```diff
@@ -14,7 +14,7 @@ Google Cloud Setup
 
 .. warning:: Google Cloud Platform must approve several requests that may take up to 1 day to complete.
 
-1. If necessary, create an account at `Google Cloud <https://cloud.google.com>`_ and create a Google Cloud project, making sure you have at least one account with the `Owner` role. Take note of the project ID (you will need this later during Kiosk configuration).
+1. If necessary, create an account at `Google Cloud <https://cloud.google.com>`_ and create a Google Cloud project, making sure you have at least one account with the `Owner` role.
 
 2. Make sure the `Kubernetes Engine API <https://console.cloud.google.com/apis/api/container.googleapis.com/overview>`_ is enabled.
```
Comment (MekWarrior) [inline docs/source/GETTING_STARTED.rst:20]: "Google does not include these GPU nodes by default as part of its free tier thus necessitating the upgrade." Thus is problematic here. It should be "free tier; thus, necessitating the upgrade." or "free tier, which is why the upgrade is required."

File: README.md [hunk 2]
```diff
@@ -6,7 +6,7 @@
...
 
-The DeepCell Kiosk is designed to allow researchers to easily deploy and scale a deep learning platform for biological image analysis. Once launched, users can drag-and-drop images to be processed in parallel using publicly available, or custom-built, TensorFlow models. To train custom models, please refer to [`deepcell-tf`](https://github.com/vanvalenlab/deepcell-tf), which was designed to facilitate model development and is capable of exporting these models for use with the DeepCell Kiosk.
+The DeepCell Kiosk is designed to allow researchers to easily deploy and scale a deep learning platform for biological image analysis. Once launched, users can drag-and-drop images to be processed in parallel using publicly available, or custom-built, TensorFlow models. To train custom models, please refer to [deepcell-tf](https://github.com/vanvalenlab/deepcell-tf), which was designed to facilitate model development and is capable of exporting these models for use with the DeepCell Kiosk.
 
 The scalability of the DeepCell Kiosk software is enabled by [Kubernetes](https://kubernetes.io/). At present, the Kiosk is only compatible with [Google Cloud](https://cloud.google.com/).
```
[the remark this answers is not in the document]
Comment (willgraf) [reply README.md:12]: > Spacing on RTD is odd here (not sure if there's anyway to fix it though).

I don't think we can do much about the spacing unless we want to merge paragraphs.

> should we change Examples to Example until we have more than one?

I think "Example" will be a clunky header. I don't see a big downside in leaving it plural.

File: docs/source/GETTING_STARTED.rst (hunk 1 above)
Comment (willgraf) [reply docs/source/GETTING_STARTED.rst:20]: Rereading this section makes we want to reshuffle the order of the sentences:

> 3. The recent success of deep learning has been critically dependent on accelerated hardware like GPUs. Similarly, the strength of the DeepCell Kiosk is its ability to recruit and scale GPU nodes based on demand. In order to add accelerated hardware to the clusters you will launch, you will need to `upgrade <https://cloud.google.com/free/docs/gcp-free-tier#how-to-upgrade>`_ your Google Cloud account.

> NOTE: The account upgrade may take some time, as Google will need to approve the upgrade. You may also need to log in and out of your account for the upgrade to take effect. Once your account is upgraded you should be able to see GPU options in `the quota panel <https://console.cloud.google.com/iam-admin/quotas>`_.

How does that sound?

Comment (MekWarrior) [reply docs/source/GETTING_STARTED.rst:20]: I like that much better than what we had. The only concern I would have (and this is true in both versions) is that the user may think that adding a GPU is optional.

**Pull request merged**